hal/lib/athena/PDP.cpp

```diff
@@ -6,14 +6,14 @@
 /*----------------------------------------------------------------------------*/
 
 #include "HAL/PDP.h"
+#include "ctre/PDP.h"
```

Comment:
This has been fixed by adding include order overriding to `.styleguide`. I think adding support for it made the Python code cleaner. :)